### ISSUE TITLE
Missing input files for rule symlinks

### ISSUE THREAD

**Issue by EugeneEA** [rajewsky-lab/spacemake#60]
Hi, I'm trying to launch run of spacemake
I've installed everything and bump into two errors:

first. `spacemake init --dropseq_tools /home/eugenea/bin/Drop-seq_tools-2.5.1 --download_species human  --download_species`
finishes snakemake part, but seems to fails later 
![image](https://user-images.githubusercontent.com/57629300/217854682-e9b6bb53-25cd-4530-a173-0b4bdb3a5fc2.png)

Second. (after re-init with the downloaded genomes `spacemake config add_species   --name homo_sapiens   --reference genome  --sequence /home/eugenea/spacemake_test/species_data/human/human_genome.fa   --annotation /home/eugenea/spacemake_test/species_data/human/human_annotation.gtf` ) Spacemake fails to build dag 
![image](https://user-images.githubusercontent.com/57629300/217855306-8f9b77bd-cb69-48e4-8ccf-9af9240fee5b.png)

project was added like in the docs:
```
spacemake projects add_sample \
   --project_id spacemake_test_run \
   --sample_id visq_test \
   --R1 /home/data/fastq/100086/scRNAseq/vis1_S1_L001_R1_001.fastq.gz /home/data/fastq/100086/scRNAseq/vis1_S1_L002_R1_001.fastq.gz \
   --R2 /home/data/fastq/100086/scRNAseq/vis1_S1_L001_R2_001.fastq.gz /home/data/fastq/100086/scRNAseq/vis1_S1_L002_R2_001.fastq.gz \
   --species homo_sapiens \
   --puck visium \
   --run_mode visium \
   --barcode_flavor visium
```

Best, Eugene

**Comment by sztankatt** [comment]
Hi @EugeneEA,

Thank you for using spacemake! Can you confirm that your spacemake version is `0.7`?

I think something got broken when upgrading from `0.5.5` to `0.7`. 

I would suggest you install the former with `pip install spacemake==0.5.5` and try these commands again.

Let me know if it works.

Best,
Tamas

**Comment by nukappa** [comment]
Hi Eugene,

I cannot reproduce the second error. Can you post here the output of `spacemake projects list` that you have after doing `spacemake init` and having added the species?

**Comment by EugeneEA** [comment]
> Hi @EugeneEA,
> 
> Thank you for using spacemake! Can you confirm that your spacemake version is `0.7`?
> 
> I think something got broken when upgrading from `0.5.5` to `0.7`.
> 
> I would suggest you install the former with `pip install spacemake==0.5.5` and try these commands again.
> 
> Let me know if it works.
> 
> Best, Tamas

Thanks, I've uninstalled version 0.7 repeat everything with version 0.0.5 and it works out.

**Comment by marvin-jens** [comment]
Hi @EugeneEA,

I think I've fixed the issue with a recent commit to master. Will be included in 0.7.3.
